blueocean-dashboard/src/main/js/components/karaoke/components/Step.jsx

```diff
@@ -77,11 +77,6 @@ export class Step extends Component {
     }
 
     render() {
-        // early out
-        if (this.pager.pending) {
-            logger.debug('pending returning null');
-            return null;
-        }
```

Comment:
ah yes, react accepts null as meaning "remove this component or don't bother rendering". Looks like left over debug code.